habitat-lab/habitat/core/env.py

```diff
@@ -111,6 +111,22 @@ def __init__(
         else:
             self.number_of_episodes = None
 
+        # Filter out inactive sensors from habitat-sim config.
```

Comment:
How about `Filter out visual sensors from the habitat-sim config that aren't used by this Gym env.`

Reply:
It would be nice to use the terms "active" and "inactive" consistently, or not introduce these terms at all. Does inactive mean not used or not rendered? What do you call a visual sensor that is unused but still rendered (i.e. if you don't have this cull flag set)?